tests/test_bytes.py

```diff
@@ -1,9 +1,13 @@
 from __future__ import annotations
 
+import typing as t
```

Comment:
Since the only import you use is `Generator`, just use `from typing import Generator`

Reply:
ack. its a habit for me.